sql_queries.py

```diff
@@ -100,12 +106,20 @@
 create_categories = "CREATE TABLE IF NOT EXISTS category(" \
                     "guild_id TEXT NOT NULL," \
                     "category_id TEXT NOT NULL," \
-                    "PRIMARY KEY (guild_id)" \
+                    "archive_id TEXT," \
+                    "PRIMARY KEY (guild_id, category_id)" \
                     ")"
```

Comment:
Why is category_id in the primary key now?

Reply:
fixed